drv/transceivers-server/src/main.rs

```diff
@@ -97,6 +97,13 @@ struct ServerImpl {
     net: task_net_api::Net,
     modules_present: LogicalPortMask,
 
+    /// The Front IO board is not guaranteed to be present and ready
+    ///
+    /// None = Front IO is not yet ready
+    /// Some(false) = Front IO is not present
+    /// Some(true) = Front IO is present and ready.
+    front_io_board_present: Option<bool>,
```

Comment:
Style nit, take it or leave it: this feels complex enough that I might consider  replacing it with something like 
```rust
enum FrontIoStatus {
   NotReady,
   NotPresent,
   Ready,
}
```

Reply:
This is a much better idea :) Thanks!